internal/workload/podman/podman.go

```diff
@@ -11,21 +11,32 @@ import (
 	api2 "github.com/jakub-dzon/k4e-device-worker/internal/workload/api"
 )
 
-type Podman struct {
+type Podman interface {
```

Comment:
no need for `//go:generate mockgen...` ?

Reply:
Added, thanks.